filebeat/input/journald/config.go

```diff
@@ -48,7 +48,13 @@ type config struct {
 	CursorSeekFallback journalread.SeekMode `config:"cursor_seek_fallback"`
 
 	// Matches store the key value pairs to match entries.
-	Matches []journalfield.Matcher `config:"include_matches"`
+	Matches journalfield.IncludeMatches `config:"include_matches"`
+	// Units stores the units to monitor
+	Units []string `config:"units"`
+	// Kernel stores whether kernel messages should be monitored
+	Kernel bool `config:"kernel"`
+	// Identifiers stores the syslog identifiers to watch
+	Identifiers []string `config:"identifiers"`
```

Comment:
I know it's  kind nitpick, but for the sake of consistency, lets add some blank lines here:
```suggestion
	Matches journalfield.IncludeMatches `config:"include_matches"`

	// Units stores the units to monitor
	Units []string `config:"units"`

	// Kernel stores whether kernel messages should be monitored
	Kernel bool `config:"kernel"`

	// Identifiers stores the syslog identifiers to watch
	Identifiers []string `config:"identifiers"`
```